From b42ba1c5dac0b0202a7dae93c02370afb611e615 Mon Sep 17 00:00:00 2001
From: Piyush Gupta <gpiyush@vmware.com>
Date: Mon, 27 Feb 2023 12:42:42 +0000
Subject: [PATCH 03/16] isoInstaller.py: Raise exception in case installer
 fails.

Fixes https://github.com/vmware/photon-os-installer/commit/ede221e30e17feb733f8327b1ba03386bc5884b5

Change-Id: I58abe505aa230dd498c2461ab926068b563e59c6
---
 photon_installer/isoInstaller.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/photon_installer/isoInstaller.py b/photon_installer/isoInstaller.py
index 12bf307..39dd415 100755
--- a/photon_installer/isoInstaller.py
+++ b/photon_installer/isoInstaller.py
@@ -87,8 +87,8 @@ class IsoInstaller(object):
 
             installer.configure(install_config, ui_config)
             installer.execute()
-        except Exception:
-            pass
+        except Exception as err:
+            raise Exception(f"Failed with error: {err}")
 
     def _load_ks_config(self, path):
         """kick start configuration"""
-- 
2.23.1

